config/environments/staging.rb

```diff
@@ -0,0 +1,69 @@
+Avalon::Application.configure do
+  # Settings specified here will take precedence over those in config/application.rb
+
+  # Code is not reloaded between requests
+  config.cache_classes = true
+
+  # Full error reports are disabled and caching is turned on
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
+
+  # Disable Rails's static asset server (Apache or nginx will already do this)
+  config.serve_static_assets = false
+
+  # Compress JavaScripts and CSS
+  config.assets.compress = false # FIXME Chris 20120507 -> set this temporarily
+
+  # Don't fallback to assets pipeline if a precompiled asset is missed
+  config.assets.compile = true # FIXME Chris 20120507 -> set this temporarily
+
+  # Generate digests for assets URLs
+  config.assets.digest = true
+
+  # Defaults to Rails.root.join("public/assets")
+  # config.assets.manifest = YOUR_PATH
+
+  # Specifies the header that your server uses for sending files
+  # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for apache
+  # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for nginx
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  # config.force_ssl = true
+
+  # See everything in the log (default is :info)
+  # config.log_level = :debug
```

Comment:
There's lots in this file that should be reviewed but the big question is this file even being used? I assume ansible is overwriting it just like we do for ERA ( :-1: )? Hopefully you have reflecting changes in ansible for this as well?